packages/@okta/vuepress-theme-prose/assets/css/prose.scss

```diff
@@ -7,6 +7,11 @@
 @import 'abstracts/colors';
 @import 'abstracts/variables';
 
+// Icons
+@import 'okta/global/fontawesome';
+@import 'okta/global/fontello';
+@import 'okta/icons/all';
```

Comment:
Part of getting rid of the stand-alone guides layout was to get rid of all the old `okta` stylesheets.  We should figure out if we ACTUALLY need these, or if we can adjust and bring these into either our standard styles, or into odyssey